perception/autoware_lidar_transfusion/lib/ros_utils.cpp

```diff
@@ -49,14 +49,14 @@ void box3DToDetectedObject(
 
   // pose and shape
   // mmdet3d yaw format to ros yaw format
-  float yaw = box3d.yaw + autoware::universe_utils::pi / 2;
+  float yaw = box3d.yaw;
   obj.kinematics.pose_with_covariance.pose.position =
```

Comment:
If we don't need any format conversion, please remove this line and upper comment as well (line 51). We can use `box3d.yaw` directly.

Reply:
FIXED